src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PreReleaseUserServiceTests.cs

```diff
@@ -489,16 +633,112 @@ await using (var userAndRolesDbContext = DbUtils.InMemoryUserAndRolesDbContext(c
                 Assert.Equal("test@test.com", result.Right.Email);
             }
 
+            await using (var context = DbUtils.InMemoryApplicationDbContext(contextId))
+            {
+                var releaseInvite = await context.UserReleaseInvites
+                    .Where(userReleaseInvite => userReleaseInvite.ReleaseId == release.Id)
+                    .SingleAsync();
+
+                Assert.Equal("test@test.com", releaseInvite.Email);
+                Assert.Equal(ReleaseRole.PrereleaseViewer, releaseInvite.Role);
+                Assert.False(releaseInvite.Accepted);
```

Comment:
I think for my cognitive load, it might be good to put in some test code comments that describe why and when we're expecting the combinations of email sent, email not sent but invite accepted, invite not accepted etc etc.  Might be that I'm having a bad brain day...